client/src/Views/Home/Prospects/Prospects.js

```diff
@@ -1,16 +1,176 @@
-import React from "react";
-import { Grid, Typography, withStyles } from "@material-ui/core";
+import React, { Component } from "react";
+import { Grid, withStyles, Typography } from "@material-ui/core";
+import { Route } from "react-router-dom";
+import ProspectSidebar from "./ProspectSidebar";
+import ProspectDashboardHeader from "./ProspectDashboardHeader";
+import Box from '@material-ui/core/Box';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { getProspectData } from "Utils/api";
+import AuthUserContext from "Components/Session/AuthUserContext";
+import styles from "Components/Table/ProspectTableStyles";
+import CloudIcon from '@material-ui/icons/Cloud';
+import ProspectTableCheckbox from 'Components/Checkbox/ProspectTableCheckbox';
 
-const styles = () => ({
-  root: {
-    flexGrow: 1
+
+class Prospects extends Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      prospects : [],
+      filteredProspects: [],
+      user : {},
```

Comment:
I think you should be able to put `this.context.user` directly into state here rather than in mount